src/select-menu/src/Option.js

```diff
@@ -22,45 +41,31 @@ const Option = memo(
       ...rest
     } = props
 
-    const { tokens } = useTheme()
-
-    const disableProps = { color: 'muted' }
-    const selectedProps = { color: tokens.selectedOptionColor }
-    const emptyProps = {}
-
-    let textProps = emptyProps
-    if (disabled) {
-      textProps = disableProps
-    }
-
-    if (isSelected) {
-      textProps = selectedProps
-    }
+    const { className: themedClassName, ...boxProps } = useStyleConfig(
+      'Option',
+      emptyObject,
+      pseudoSelectors,
+      internalStyles
+    )
 
     return (
       <TableRow
```

Comment:
Is it worth breaking apart from `<TableRow />` or is that going to come in a follow-up PR?

Reply:
In a follow up